New Homepage Design (progress)

emersonthis opened the pull request:
Mostly addresses #146 

There's very little design effort in this (yet). Mostly trying to make sure I understood the content requirements first. I also did not refactor the header/navigation yet...

<img width="1573" alt="Screen Shot 2020-03-21 at 10 22 45 AM" src="https://user-images.githubusercontent.com/1728139/77232389-fed93600-6b5d-11ea-9023-29e3d38b17ed.png">

This PR also includes some reorganizing of the CSS files.

nditada commented:
It looks great so far, it will definitely need some GX but the structure is good.

emersonthis commented:
@nditada I updated this PR with my interpretation of what I think is the intended design. I simplified the curve graphic because full version was very "busy" behind the text.

emersonthis commented:
@nditada The design in the issue also calls for some changes to the navigation. I can do that also, but I think we should focus on getting this merged as soon as possible because it includes some CSS reorganization which will result in lots of merge conflicts if this sits for too long.
